src/containers/CreateNote/CreateNote.js

```diff
@@ -67,12 +66,17 @@ class CreateNote extends React.Component {
     return currentList;
   }
 
-  handleSubmit = event => {
+  handleSubmit = (event) => {
     event.preventDefault();
-    // const { addNewNote } = this.props;
     const { title, noteItems } = this.state;
-    this.props.postNote({ title, noteItems, id: uuid() });
-    // addNewNote({ title, noteItems });
+    const { id: noteId } = this.props.currentNote;
+    if (noteId) {
+      this.props.putNote({ title, noteItems, id: noteId});
+      this.props.setCurrentNote({});
+      this.props.showPopUp(false);
```

Comment:
This is really good! I like the way you found to reuse this component! Great work! 😄